### ISSUE TITLE
A script engine & Qt 5.6

### ISSUE THREAD

**Issue by Paliha** [x64dbg/x64dbg#1085]
Hi! On the forum https://exelab.ru in personal account, a link to the archive and password

**Comment by mrexodia** [comment]
`C:\Users\Admin\Downloads\Bronco_Bin64.rar`

**Comment by Paliha** [comment]
Ok!
Just out of interest, have used the delay in the script:
@sleep:
cmp sleep2,0x1000
je @log
add sleep2,1
jmp @sleep
@log:
log "-----------"
Something to think about...))))

**Comment by mrexodia** [comment]
Yes, Sleep is indeed used. This is to not make things go crazy when waiting
for the debugger to be paused again.
